src/Commands/UserCommands/WeatherCommand.php

```diff
@@ -37,7 +37,8 @@ class WeatherCommand extends UserCommand
      */
     private function getWeather($location)
     {
-        $url = 'http://api.openweathermap.org/data/2.5/weather?q=' . $location . '&units=metric';
+        $appid = 'Your_APIkey_from_openweathermap.org';
```

Comment:
You can't ash  user  to edit file inside the  library you should set it with: https://github.com/akalongman/php-telegram-bot#commands-configuration